### ISSUE TITLE
Feature Request: Gray/White pencil option 

### ISSUE THREAD

**Issue by catflint** [ddvk/remarkable-hacks#241]
Would love the option to use white and gray with the pencil so I can use it as a textured eraser. Thanks

**Comment by Baccanno** [comment]
I dig into this !

**Comment by dalistylin** [comment]
> 
> 
> Would love the option to use white and gray with the pencil so I can use it as a textured eraser. A grayscale palette in the tool sections would be great too. Something like: Light Gray, Medium Gray and Dark Gray. Thanks.

I agree. For drawing, the option for grey and white for pencil and other pen types that don't have them normally available would be great!

**Comment by tzcone** [comment]
+1

**Comment by ddvk** [comment]
i dont know if this is easily doable.  afaik, the pencil uses some bitmap patterns...

**Comment by Baccanno** [comment]
> i dont know if this is easily doable. afaik, the pencil uses some bitmap patterns...

The pencil uses only black, solely the density of the dots pattern makes it more or less greyish. Wouldn't you be able to add the same bitmap pattern with gray and white? Or could you shift the greyscale value of the pattern ?

Still thanks for the answer, if you need any help with the image processing part, feel free to ask

**Comment by dstango** [comment]
@Baccano: don't overestimate the scope of the hacks ;-) ... afaik the hacks so far only tweak some parameters for stroke thickness etc. and leave the actual rendering process untouched. If the rendering will - maybe accidentally - process some "color setting" (black, white, grey) the implementation might be more or less easily feasible. The lines-File-format (https://remarkablewiki.com/tech/filesystem#lines_file_format) actually seems to always have 4 bytes for the stroke color -- so we might be lucky ;-) ...
Yet we don't know if the color will be considered (and we don't know what's happening when exporting data).

**Comment by catflint** [comment]
@ddvk the paintbrush tool seems to use a texture too. are there any similarities in code between it and the pencil that can be referenced?
